dspace-server-webapp/src/main/java/org/dspace/app/rest/scripts/handler/impl/RestDSpaceRunnableHandler.java

```diff
@@ -304,4 +309,24 @@ private void addLogBitstreamToProcess(Context context) throws SQLException, IOEx
             log.error("RestDSpaceRunnableHandler with process: " + processId + " could not write log to process", e);
         }
     }
+
+    @Override
+    public List<UUID> getSpecialGroups() {
+        Context context = new Context();
+        List<UUID> specialGroups = new ArrayList<>();
+        try {
+            Process process = processService.find(context, processId);
+            for (Group group : process.getGroups()) {
+                context.setSpecialGroup(group.getID());
```

Comment:
I'm not understanding why this method is saving these Groups to a brand new Context & then aborting the Context below?  Isn't that just a no-op?    It looks, to me, like this `context.setSpecialGroups()` call can be removed. If this really is necessary then we need JavaDocs / comments to describe what is happening here.

Reply:
I agree with @tdonohue here, it doesn't seem to be necessary